dev_environment/Vagrantfile

```diff
@@ -25,6 +25,24 @@ Vagrant.configure("2") do |config|
     end
 
     l3af.vm.box = "ubuntu/focal64"
+    if cfg['host_distro_codename'] == "jammy"
+        config.vm.provision "shell", privileged: false, inline: <<-EOF
+          echo "Vagrant Box jammy provisioned!"
+        EOF
+      l3af.vm.box = "ubuntu/jammy64"
+    else
+       l3af.vm.provision "shell", inline: <<-SHELL
+          wget https://raw.githubusercontent.com/pimlie/ubuntu-mainline-kernel.sh/master/ubuntu-mainline-kernel.sh
+          chmod +x ubuntu-mainline-kernel.sh
+          sudo mv ubuntu-mainline-kernel.sh /usr/local/bin/
+          sudo ubuntu-mainline-kernel.sh -i 5.15.0
+          apt --fix-broken -y install
+        SHELL
+
+        # restart after upgrade
+        l3af.vm.provision :reload
+    end
+
```

Comment:
Is this forcing to download 5.15 kernel to /usr/local/bin/ if focal distro detected?  Maybe it is good idea to put a comment says, need 5.15 kernel if none jammy distro detected since the ebpf development now require 5.15.0 kernel.